### DISCUSSION TITLE
Stop plotting in 11precent

### DISCUSSION THREAD

**Discussion by Bgheri1368** [Chia-Network/chia-blockchain#6506] [Support]
Hi , i have tried many times to plat , but it used to stop on 11 % . 
all setting was on default mode ,
temporary drive ssd 500 G
cpu i5 8TH
ram 4 
what is the problem?
![WhatsApp Image 2021-06-06 at 13 52 48](https://user-images.githubusercontent.com/85434852/120919396-97f65d00-c6ce-11eb-98bf-d0f25b69704f.jpeg)

**Comment by gromran** [comment]
What is there about this in the log file?
But I suspect "ram 4" means 4GB, which is far too few.

**Reply by Bgheri1368** [reply]
yeah its mean 4G Ram
your right, it works but it takes so longtime

**Comment by BrandtH22** [comment]
@Bgheri1368 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
